Bisulfite_QC_bisulfiteconversion.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+## make sure the following is in PATH
+## biscuit samtools, bedtools, awk
+
+function biscuitQC
+
+{
+  ## simple test to make sure all is available
+  if [[ `which biscuit 2>&1 >/dev/null` ]]; then echo "biscuit does not exist in PATH"; exit 1; fi
+  if [[ `which samtools 2>&1 >/dev/null` ]]; then echo "samtools does not exist in PATH"; exit 1; fi
+  if [[ `which bedtools 2>&1 >/dev/null` ]]; then echo "bedtools does not exist in PATH"; exit 1; fi
+  if [[ `which awk 2>&1 >/dev/null` ]]; then echo "awk does not exist in PATH"; exit 1; fi
+  for var in BISCUIT_CPGBED BISCUIT_CGIBED BISCUIT_RMSK BISCUIT_EXON BISCUIT_GENE BISCUIT_TOPGC_BED BISCUIT_BOTGC_BED input_bam input_vcf; do
+    if [[ ${!var} != "<unset>" && ! -f ${!var} ]]; then
+      >&2 echo "$var: ${!var} does not exist."
+      exit 1;
+    fi
+  done
+
+  echo "Running."
+  set -xe
+
+  ##########################
+  ## bisulfite conversion
+  ##########################
+  [[ ! -f "$input_vcf" ]] && BISCUIT_QC_BSCONV=false
+  if [[ "$BISCUIT_QC_BSCONV" == true ]]; then
+    >&2 echo "`date`---- BISCUIT_QC_BSCONV ----"
+    echo -e "BISCUITqc Conversion Rate by Base Average Table" >totalBaseConversionRate.txt
+    biscuit vcf2bed -et c $input_vcf | awk '{beta_sum[$6]+=$8; beta_cnt[$6]+=1;} END{print "CA\tCC\tCG\tCT"; print beta_sum["CA"]/beta_cnt["CA"]"\t"beta_sum["CC"]/beta_cnt["CC"]"\t"beta_sum["CG"]/beta_cnt["CG"]"\t"beta_sum["CT"]/beta_cnt["CT"];}' >>totalBaseConversionRate.txt
+
+    echo -e "BISCUITqc Conversion Rate by Read Average Table" >totalReadConversionRate.txt
+    samtools view -hq 40 $input_bam | biscuit bsconv -b $BISCUIT_REFERENCE - | awk '{for(i=1;i<=8;++i) a[i]+=$i;}END{print "CpA\tCpC\tCpG\tCpT"; print a[1]/(a[1]+a[2])"\t"a[3]/(a[3]+a[4])"\t"a[5]/(a[5]+a[6])"\t"a[7]/(a[7]+a[8]);}' >>totalReadConversionRate.txt
```

Comment:
This line and a few others limit to base quality 40. That's probably fine, just want to make sure it was intentional!